pkg/plugin/admissioncontrollers/webhook/admission_controller.go

```diff
@@ -149,16 +156,9 @@ func updateLabels(pod *v1.Pod, patch []patchOperation) []patchOperation {
 	if _, ok := existingLabels[common.SparkLabelAppID]; !ok {
 		if _, ok := existingLabels[common.LabelApplicationID]; !ok {
 			// if app id not exist, generate one
-			// pod's name can be generated, if name is not explicitly specified
-			// look for generateName instead
-			podName := "unknown"
-			if pod.Name != "" {
-				podName = pod.Name
-			} else if pod.GenerateName != "" {
-				podName = pod.GenerateName
-			}
-
-			generatedID := generateAppID(podName)
+			// for each namespace, we group unnamed pods to one single app
+			// application ID convention: __${NAMESPACE}_app_default
```

Comment:
This does not line up with the test and what we generate a `prefix`-${NAMESPACE}-`suffix`

Reply:
Fixed